### PR TITLE
fix for generating `pb.gw.go` file for query service

### PR DESCRIPTION
**Pull request by crodriguezvega**

### CODE REVIEW

File: proto/intertx/query.proto [hunk 1]
```diff
@@ -5,11 +5,14 @@ package intertx;
...
 
 import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
 
 // Query defines the gRPC querier service.
 service Query {
   // QueryInterchainAccountFromAddress returns the interchain account for given owner address on a given connection pair
-  rpc InterchainAccountFromAddress(QueryInterchainAccountFromAddressRequest) returns (QueryInterchainAccountFromAddressResponse);
+  rpc InterchainAccountFromAddress(QueryInterchainAccountFromAddressRequest) returns (QueryInterchainAccountFromAddressResponse) {
+    option (google.api.http).get = "/inter-tx/v1beta1/interchain_account_from_address/{owner}/{connection_id}";
```
Comment (damiannolan) [inline proto/intertx/query.proto:14]: Thoughts on removing `FromAddress` from this? 
We could do it in another PR.

**Review by damiannolan — APPROVED** [review]
Excellent stuff! Thanks @crodriguezvega

File: proto/intertx/query.proto (hunk 1 above)
Comment (damiannolan) [reply proto/intertx/query.proto:14]: I think for `vtbeta1` the `query.proto` file should be in a directory `proto/intertx/v1beta1/query.proto` and proto package should be updated.

Otherwise you could omit the `v1beta1` in the URL.

**Review by seantking — APPROVED** [review]
